msteams-platform/bots/how-to/conversations/Workflow bot in Teams.md

```diff
@@ -0,0 +1,460 @@
+
+The following diagram illustrates how to respond to an Adaptive Card action with TeamsFx SDK:
+
+:::image type="content" source="../../../assets/images/sbs-workflow-bot/sbs-workflow-bot-action-card.png" alt-text="workflow bot card action handler diagram" lightbox="../../../assets/images/sbs-workflow-bot/sbs-workflow-bot-action-card.png":::
+
+* **Action card**: The card where you define your action that users can invoke (DoStuff button).
+* **Card action handler**: Triggered when users invoke the corresponding card action (its `triggerVerb` is same as the `verb` property in Adaptive Card action). It can send a response card to respond to the action.
+* **Response card**: The card which respond to the action when user invokes it from the action card.
+
+To handle card actions with TeamsFx SDK, each card action handler must implement `TeamsFxAdaptiveCardActionHandler` interface. This is the interface definition for `TeamsFxAdaptiveCardActionHandler`:
+
+``` Export interface
+
+TeamsFxAdaptiveCardActionHandler 
+{
+    /**
+     * The verb defined in adaptive card action that can trigger this handler.
+     */
+    triggerVerb: string;
+
+    /**
+     * Specify the behavior for how the card response will be sent in Teams conversation.
+     * The default value is `AdaptiveCardResponse.ReplaceForInteractor`, which means the card
+     * response will replace the current one only for the interactor.
+     */
+    adaptiveCardResponse?: AdaptiveCardResponse;
+    
+    /**
+     * The handler function that will be invoked when the action is fired.
+     * @param context The turn context.
+     * @param actionData The contextual data that associated with the action.
+     */
+    handleActionInvoked(context: TurnContext, actionData: any): Promise<InvokeResponse>;
+}
+```
+
+## Customize initialization
+
+The default initialization is located in `bot/src/internal/initialize.js(ts)`.
+
+You can update the initialization logic to:
+
+1. Set `options.adapter` to use your own `BotFrameworkAdapter`.
+1. Set `options.command.commands` to include more command handlers.
+1. Set `options.cardAction.actions` to include more action handlers.
+1. Set `options.{feature}.enabled` to enable more `ConversationBot` functionality.
+
+For more information on initialization customization, see [additional initialization customization](https://github.com/OfficeDev/TeamsFx/wiki/Respond-to-chat-commands-in-Teams#customize-initialization)
+
+## Add card action
+
+Steps you can follow to add card actions:
+
+1. [Add an action to your Adaptive Card](#add-an-action-to-your-adaptive-card)
+1. [Add Adaptive Card for action response](#add-adaptive-card-for-action-response)
+1. [Add action handler](#add-action-handler)
+1. [Register the action handler](#register-the-action-handler)
+
+### Add an action to your Adaptive Card
+
```

Comment:
Could you please use the updated code snippets in this file: 
* JS: https://github.com/OfficeDev/TeamsFx/tree/dev/templates/bot/js/workflow#readme
* TS: https://github.com/OfficeDev/TeamsFx/tree/dev/templates/bot/ts/workflow#readme

Reply:
Do you want all 4 code snippets to be replaced for the heading "Add card action"?

Reply:
Yes. All most the same code, but we refined it a little bit in README and haven't updated in Github WIKI yet.

Reply:
No worries, will do.

Reply:
The code snippets have been changed with the trigger verb DoSomething, however the button shows as DoStuff is that fine? Kindly confirm.

Reply:
Yeah that's fine, in this section we are guiding developers to add a new button called `DoSomething` and respond to it with a new card.